package.json

```diff
@@ -21,25 +21,25 @@
     "@types/create-hash": "^1.2.2",
     "@types/lodash": "^4.14.136",
     "@types/mocha": "^9.0.0",
-    "@types/node": "^17.0.14",
+    "@types/node": "^12.12.31",
```

Comment:
Why is this being downgraded?

Reply:
`@types/node` should be in sync with the oldest version of node that we support (which is currently 12). https://stackoverflow.com/questions/42035263/relationship-between-the-version-of-node-js-and-the-version-of-types-node

Reply:
I also forgot until just now that we're deprecating support for node 12 since it's no longer being supported. (So moving the types to v14)